src/rshim.h

```diff
@@ -408,6 +408,10 @@ struct rshim_backend {
   /* Up to two VLAN IDs for PXE purpose. */
   uint16_t vlan[2];
 
+  /* Secure NIC mode Management. No RSHIM HW access */
+  bool locked_mode;
+  bool first_update_done;
```

Comment:
This line seems not necessary and the name of this variable causes confusion too.